packages/pages/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import classNames from "classnames";
+import { BreadcrumbsProps } from "./types.js";
+import { Link } from "../link/link.js";
+import { cta } from "../link/methods.js";
+import "./breadcrumbs.css";
+
+export interface BreadcrumbsPropsDefault extends BreadcrumbsProps {
```

Comment:
What's the value of `BreadcrumbsPropsDefault`? Also can you extend an `HTMLDivElement` and pass the props to the wrapping element like in [address](https://github.com/yext/pages/blob/main/packages/pages/src/components/address/address.tsx#L39) (...props)

Reply:
HTMLNavElement

Reply:
The implementation of this component involved two ways of using it:
1. Using `BreadcrumbsPropsDefault` and passing an array of [string, string] to manually set the breadcrumbs
2. Using `BreadcrumbsPropsStreams` and passing the profile field (?) and it would automatically unwrap the breadcrumbs from the correct directoryManager custom field